docker/rootfs/etc/s6-overlay/s6-rc.d/go2rtc-healthcheck/run

```diff
@@ -0,0 +1,16 @@
+#!/command/with-contenv bash
+# shellcheck shell=bash
+# Start the go2rtc healthcheck service
+
+set -o errexit -o nounset -o pipefail
+
+echo "[INFO] Starting go2rtc healthcheck service..." >&2
+
+while sleep 1m; do
+    # Check if the service is running
+    if ! curl -fsSL http://127.0.0.1:1984/api/streams >/dev/null; then
```

Comment:
Is there a way to set a timeout here? I think we want specific behavior for how long curl waits until it considers it didn't get a response

Reply:
Yes, good point.